package.json

```diff
@@ -9,6 +9,7 @@
   },
   "devDependencies": {
     "browserify": "^13.0.0",
+    "elliptic": "^6.5.3",
```

Comment:
Any good reason you put this pkg in package.json file?

Reply:
yes, but I am open to other suggestions.

Reply:
Yeah, it's pretty common to add a package here and then let this version satisfy where else it's a dependency